snap/snapcraft.yaml

```
@@ -23,11 +23,9 @@ parts:
     stage-packages:
       - libgl1-mesa-dri
       - va-driver-all
-    override-build: |
-      snapcraftctl build
-      mkdir -p $SNAPCRAFT_PART_INSTALL/egl/lib
-      mv $SNAPCRAFT_PART_INSTALL/usr/lib/${SNAPCRAFT_ARCH_TRIPLET}/dri $SNAPCRAFT_PART_INSTALL/egl
-      mv $SNAPCRAFT_PART_INSTALL/usr/lib/${SNAPCRAFT_ARCH_TRIPLET}/* $SNAPCRAFT_PART_INSTALL/egl/lib
+    organize:
+      'usr/lib/${SNAPCRAFT_ARCH_TRIPLET}/dri/*': egl/dri/
```

Could this just be that?

```suggestion
      'usr/lib/${SNAPCRAFT_ARCH_TRIPLET}/dri': egl/
```

---

Looks plausible but:
```
Failed to copy '/root/stage/egl/dri': no such file or directory.
Check the path and try again.
```

---

It could be:
```
      'usr/lib/${SNAPCRAFT_ARCH_TRIPLET}/dri': egl/dri
```
I think the proposed incantation marginally clearer. (But won't argue over it.)